terraform/10-account/iam.terraform-role.tf

```diff
@@ -0,0 +1,16 @@
+module "iam_terraform_role" {
+  source      = "terraform-aws-modules/iam/aws//modules/iam-assumable-role"
+  version     = "5.44.0"
+  create_role = true
+
+  role_name         = "TerraformOperator"
```

Comment:
The current handrolled role is called `Terraform` so used this name to make the distinction clear